Fixed Badge occupying space when it is anchored to a child component (#2714)
jpmorganchase/salt-ds: .changeset/bright-masks-learn.md

---
"@salt-ds/lab": patch
---

> Fixed Badge occupying space by changing from relative to absolute positioning

Reviewer: ```suggestion
Fixed Badge occupying space when it is anchored to a child component.
```
